include/crow/middlewares/cookie_parser.h

```diff
@@ -50,6 +50,12 @@ namespace crow
                 value_ = std::forward<U>(value);
             }
 
+            Cookie(const std::string& key):
```

Comment:
Maybe just mix it into one constructor with default parameter's value?

Reply:
Better not to, because of the template. However we can reuse it, thats a good point